standard/variables.md

```diff
@@ -110,6 +108,19 @@ The following definite assignment rules apply to output parameters.
 
 Within an instance constructor of a struct type, the `this` keyword behaves exactly as an output or reference parameter of the struct type, depending on whether the constructor declaration includes a constructor initializer ([§11.7.12](expressions.md#11712-this-access)).
 
+### §input-parameters-new-clause Input parameters
+
+A parameter declared with an `in` modifier is an ***input parameter***.
+
+An input parameter does not create a new storage location. Instead, an input parameter represents the same storage location as the variable given as the argument in the function member, anonymous function, or local function invocation. Thus, the value of an input parameter is always the same as the underlying variable.
+
+The following definite assignment rules apply to input parameters.
+
+- A variable shall be definitely assigned ([§9.4](variables.md#94-definite-assignment)) before it can be passed as an input parameter in a function member or delegate invocation.
+- Within a function member, anonymous function, or local function an input parameter is considered initially assigned.
+
+For a `struct` type, within an instance method or instance accessor ([§11.2.1](expressions.md#1121-general)) or instance constructor with a constructor initializer, the `this` keyword behaves exactly as an input parameter of the struct type ([§11.7.12](expressions.md#11712-this-access)).
```

Comment:
What does this paragraph mean; can you show an example?

Reply:
I removed this paragraph, but made the correct updates to §12.8.13